robotidy/transformers.py

```diff
@@ -0,0 +1,38 @@
+import inspect
+import sys
+
+from robot.parsing import ModelTransformer
+
+
+def load_transformers(modes):
+    """ Dynamically load all classess from this file with attribute `name` defined in modes """
+    transfomers = []
+    classes = inspect.getmembers(sys.modules[__name__], inspect.isclass)
```

Comment:
Wouldn't `globals()` work here as well?

Reply:
Yes it would work - but we would need to filter out the list for non classes and not transformers. Right now I'm not sure what's the best - we could probably even skip the whole dynamic loading in favour of hard coded names. But this method will come handy when implementing plugin like functionality for tidy (custom transformers). 

When I think about this, inspect module will work better than globals if we're going to call this method on external files (ie. `robotidy --load-transformers-from custom_transformers.py`)

Reply:
I don't see much benefits with dynamic loading. Explicitly listing internally used transformers would be more straightforward.

I would prefer listing custom transformers one-by-one instead of loading a bunch of them from a module. That usage could be something like `--transformer CustomTransformer.py` or `--transformer module.ClassName` similarly as you can take e.g. pre-run modifiers and listeners into use with RF itself.